controllers/Index.php

```diff
@@ -228,50 +227,53 @@ private function loadItems($options, $tags) {
 
         $canUpdate = \F3::get('allow_public_update_access') == 1
                         || $_SERVER['REMOTE_ADDR'] === $_SERVER['SERVER_ADDR']
-                        || $_SERVER['REMOTE_ADDR'] === "127.0.0.1"
+                        || $_SERVER['REMOTE_ADDR'] === '127.0.0.1'
                         || \F3::get('auth')->isLoggedin() == 1;
         if ($options['source'] && $canUpdate) {
             $itemsHtml = '<button type="button" id="refresh-source" class="refresh-source">' . \F3::get('lang_source_refresh') . '</button>';
         }
 
-        foreach($itemDao->get($options) as $item) {
+        foreach ($itemDao->get($options) as $item) {
             // parse tags and assign tag colors
-            $itemsTags = explode(",",$item['tags']);
+            $itemsTags = explode(',', $item['tags']);
             $item['tags'] = array();
-            foreach($itemsTags as $tag) {
+            foreach ($itemsTags as $tag) {
                 $tag = trim($tag);
-                if(strlen($tag)>0 && isset($tagColors[$tag]))
+                if (strlen($tag) > 0 && isset($tagColors[$tag])) {
                     $item['tags'][$tag] = $tagColors[$tag];
+                }
             }
-            
+
             $this->view->item = $item;
             $itemsHtml .= $this->view->render('templates/item.phtml');
         }
 
-        if(strlen($itemsHtml)==0) {
-            $itemsHtml = '<div class="stream-empty">'. \F3::get('lang_no_entries').'</div>';
+        if (strlen($itemsHtml) == 0) {
+            $itemsHtml = '<div class="stream-empty">' . \F3::get('lang_no_entries') . '</div>';
         } else {
-            if($itemDao->hasMore())
-                $itemsHtml .= '<div class="stream-more"><span>'. \F3::get('lang_more').'</span></div>';
-                $itemsHtml .= '<div class="mark-these-read"><span>'. \F3::get('lang_markread').'</span></div>';
```

Comment:
No indention is wrong here: the button should be shown even if there are no more items to show, because it is intended to mark visible items as read.

Note: this code is removed by #846.